### DISCUSSION TITLE
Getting the 'Not Found' screen

### DISCUSSION THREAD

**Discussion by dbaranger** [adamghill/fediview#16] [Q&A] [answered]
I'm seeing the 'Not Found' screen when I click 'Follow'.  For example, the Follow button opens this link: https://fediview.com/fediscience.org/@khinsen@scholar.social, but if I remove 'fediview.com/' then it correctly opens. I've replicated this in both Firefox and Chrome. Not sure what this could be due to - does anyone have any suggestions for debugging?

**Comment by adamghill** [comment]
I'm guessing this is a bug, thanks for reporting it! I'll try to get it fixed asap.

**Reply by adamghill** [reply, marked as answer]
Should be fixed in https://github.com/adamghill/fediview/commit/fc8dd1bd77977deab3e681f2695762a88cf11155 which is going live right now.

**Reply by dbaranger** [reply]
Brilliant, thanks!
